CHANGELOG.md

```diff
@@ -1,7 +1,36 @@
 # Change Log
 
-Details changes in each release of EarlGrey. EarlGrey follows
-[semantic versioning](http://semver.org/).
+Details changes in each release of EarlGrey. EarlGrey follows [semantic versioning](http://semver.org/).
+
+## [1.8.0](https://github.com/google/EarlGrey/tree/1.8.0) (03/17/2017)
+
+```
+Baseline: [0dc7c18]
+   + [0dc7c18]: 1.8.0 Release
+```
+
+### New Features
+* Added multi-finger swipe action API's - grey_multiFingerSwipe{Fast|Slow}InDirection, 
+  grey_multiFingerSwipe{Fast|Slow}InDirectionWithStartPoint.
+
+### Bug Fixes
+* Fixed issue with accessibility spamming "Remote service does not respond to _accessibilityMachPort" message on iOS 9.1 device.
+* Fixed issues with EarlGrey working with a `UIAccessibilityTextFieldElement`.
+* Fixed typing by blacklisting `UICompatibilityInputViewController` in UIViewController tracking.
+
+### Compatibility
+* Requires iOS 8 as the minimum deployment target.
+* Supports Xcode 8.2.1 and iOS 10.2.1 on devices and simulators.
+* The EarlGrey gem runs out of the box for Swift 3.0 and Swift 2.3.
+
+### Enhancements
+* Improvements to the EarlGrey FunctionalTests TestRig.
+* All GREYAssertXXX macros wait for now wait for the app to idle before being evaluated.
+* Unified the Copy Files modification script for Carthage and CocoaPods support.
+
+### Contributors
+[bootstraponline](https://github.com/bootstraponline)
```

Comment:
still not 1 per line. see preview:
https://github.com/tirodkar/EarlGrey/blob/60f4aaf2543a219174e44f6d6cc6c90b7947d4d1/CHANGELOG.md